tools/appliance/systemvmtemplate/scripts/install_systemvm_packages.sh

```diff
@@ -78,11 +78,13 @@ function install_packages() {
   apt-get clean
   apt-get autoclean
 
-  #32 bit architecture support:: not required for 32 bit template
+  ${apt_get} install links
```

Comment:
For 64-bit binaries, this may be moved to lines up like line 60-73. I'm not sure where links is used at all (perhaps for debugging purposes).

Reply:
change if you will